### ISSUE TITLE
[Bug] Harvesting Effectiveness 

### ISSUE THREAD

**Issue by WyattLiu** [Chia-Network/chia-blockchain#16169]
### What happened?

For me, with 2 harvesters, the estimation in the question mark seems based on the main local harvester.

### Version

2.0.0

### What platform are you using?

Linux

### What ui mode are you using?

GUI

**Comment by wjblanke** [comment]
Do you mean the information is only being calculated from one of your harvesters? Can you send us a screenshot with the problem? do you have the correct number of plots appearing? let us know!

**Comment by WyattLiu** [comment]
[cid:c4a4059e-7da3-4f4e-baeb-f1a9bf1f4647]
It is a minor issue, jus the (?) informational message is using peer to estimate signage point count... I can get a good destination from the pool.

Thank you for looking into this
________________________________

**Comment by WyattLiu** [comment]
![8C2D7F1B-51F9-44F3-9673-A283E998364C_1_201_a](https://github.com/Chia-Network/chia-blockchain/assets/17634602/5c265667-b257-48f4-aee9-7b3aa8b11fe7)

**Comment by WyattLiu** [comment]
The latency is a bit off as it is doing live replots...

**Comment by emlowe** [comment]
You are talking specifically about the "tooltip" information graph if you hover/click on the question mark - correct?

The text in the dark grey box is inaccurate because you have 6137 plots and the text suggests you have 3586 plots

**Comment by WyattLiu** [comment]
Yeah, just the tooltip. It is a minor issue, would be nice if this is corrected.
________________________________

**Comment by wjblanke** [comment]
Izumi can u look at this please. U can choose to make it per harvester if we move the tooltip

**Comment by ChiaMineJP** [comment]
I'll take a look

**Comment by paninaro** [comment]
Should be addressed in builds > 2.1.0-rc1
